### PR TITLE
Add an `edge` tag for opa docker images

### PR DESCRIPTION
**Pull request by patrick-east**
This tag will follow the current master branch and lines up with what
is in the docs as the `edge` version alias.

Signed-off-by: Patrick East <east.patrick@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -100,15 +100,30 @@ push-latest:
...
+	docker push $(IMAGE):edge
+	docker push $(IMAGE):edge-debug
+	docker push $(IMAGE):edge-rootless
+
 .PHONY: docker-login
 docker-login:
 	@docker login -u ${DOCKER_USER} -p ${DOCKER_PASSWORD}
 
+.PHONY: image-full
+image-full: docker-login image-quick push
```
Comment (tsandall) [inline Makefile]: Nit: this is super minor but I'd rather call the `image-full` target something like `push-image` to indicate that it's going to try to push. The other `image` and `image-quick` targets don't do any pushing (i.e., they just build locally).

Comment (patrick-east) [reply Makefile]: 👍 updated in latest changes

**Comment by tsandall** [comment]
@patrick-east if you fix the conflict we can merge this.

**Comment by patrick-east** [comment]
Rebased and fixed conflicts, should be good to go.

**Pull request merged**